src/config.ps1

```diff
@@ -0,0 +1,95 @@
+function ConvertTo-HashTable {
+	param (
+		[Parameter(ValueFromPipeline)]
+		[PSCustomObject]$Object
+	)
+	$Table = @{}
+	$Object.PSObject.Properties | ForEach-Object {
+		$V = $_.Value
+		if ($V -is [Array]) {
+			$V = [System.Collections.ArrayList]$V
+		} elseif ($V -is [PSCustomObject]) {
+			$V = ($V | ConvertTo-HashTable)
+		}
+		$Table.($_.Name) = $V
+	}
+	return $Table
+}
+
+function GetPwrHome {
+	if ($PwrHome) {
+		$PwrHome
+	} elseif ($env:PwrHome) {
+		$env:PwrHome
+	} else {
+		"$env:LocalAppData\pwr"
```

Comment:
Since we're moving this location, should we take the opportunity to rename to `$env:LocalAppData\Airpower`?